src/ecs/fargate/environment.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed
+ * under the Apache License Version 2.0.
+ *
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2021 Datadog, Inc.
+ */
+
+import { FargateDefaultEnvVars } from "./constants";
+import { DatadogECSFargateProps } from "./interfaces";
+import { EnvVarManager } from "../environment";
+
+export class FargateEnvVarManager extends EnvVarManager {
+  constructor(props: DatadogECSFargateProps) {
+    super(FargateDefaultEnvVars);
+
+    this.addAll(props.environmentVariables);
+
+    this.add("DD_API_KEY", props.apiKey);
+    this.add("DD_SITE", props.site);
+    this.add("DD_ENV", props.env);
+    this.add("DD_SERVICE", props.service);
+    this.add("DD_VERSION", props.version);
+    if (props.globalTags && this.retrieve("DD_TAGS")) {
+      console.debug(`Global tags are set in both the environment variable
+        and the props. The environment variable will be overwritten.`);
```

Comment:
This will print some spaces on the second line. You can either use `+` to concat the strings or remove the spaces.